packages/edit-site/src/components/page/style.scss

```diff
@@ -33,6 +32,10 @@
 }
 
 .edit-site-page-content {
-	padding: $grid-unit-40 $grid-unit-40;
-	overflow-x: auto;
+	height: 100%;
+	display: flex;
+	overflow: auto;
+	flex-flow: column;
+	position: relative;
+	z-index: z-index(".edit-site-page-content");
```

Comment:
Without this style, the `.edit-site-layout__sidebar-region` would overlap on top of the `.edit-site-page` in the mobile layout, making some buttons impossible to press.